CleverTap/Runtime/Native/UnityNativeWrapper/Models/UnityNativeMetadataResponseInterceptor.cs

```diff
@@ -0,0 +1,46 @@
+using System;
+using System.Collections.Generic;
+using CleverTapSDK.Native;
+using CleverTapSDK.Utilities;
+
+#if (!UNITY_IOS && !UNITY_ANDROID) || UNITY_EDITOR
+namespace Native.UnityNativeWrapper
```

Comment:
Fix namespace to be CleverTapSDK.Native and remove it from the usings.

Reply:
Done